### ISSUE TITLE
When in navigation mode, escape key should select parent

### ISSUE THREAD

**Issue by vg-robin** [WordPress/gutenberg#17408]
Selecting parent blocks has always been a pain. Click-through mode kinda works for devs that know the current block structure, but my boss hates it, and thinks our clients will to. (Can't disagree, and if it stays it certainly needs to be made a user-level setting.)

Navigation mode works fine, but once in nav mode, why not make another press of the escape key select the parent block? Once in nav mode, it isn't used for anything else (as far as I can see).

I propose this is done regardless of what happens with breadcrumbs and click through and any parallel solutions.

Unless I'm missing something, wouldn't this be a quick win to a much needed (at least by me) shortcut?

(Additional suggestion: if the left and right arrow keys aren't being used in nav mode, why not map them the same as up and down? Then it's far more intuitive for selecting side-by-side blocks, like columns.)

**Comment by HardeepAsrani** [comment]
I personally feel that Left/Right keys can be used to jump from child to parent instead, as Escape is perfect to just close the navigation popup.

**Comment by vg-robin** [comment]
Hi Hardeep. Thanks for commenting.
Yes, I'd be happy with that alternative - using Escape to leave Nav mode would work fine, and would be intuitive, though it duplicates the existing Tab key function in that context. Using left and right navigating to "parent" and "first child" respectively would work fine. (Hadn't considered navigating to children of the current block, but that's certainly needed, and Tabbing doesn't cover it.)

**Comment by reageek** [comment]
Escape is the standard for exiting out / going back a level. For navigating into children, if it were to follow the normal accessibility patterns it should be space bar to go into the the child nav, tab and shift tab to move forward and backwards through the items.

**Comment by youknowriad** [comment]
I agree with @HardeepAsrani here I think "ArrowLeft/Right" or "Up/Down" shouldn't go straight to the next/previous block if the current block contains innner blocks, it should first navigate throught the hierarchy. This keeps the interaction simple enough.

**Comment by talldan** [comment]
This was solved by https://github.com/WordPress/gutenberg/pull/22614, since Gutenberg 8.3 left and right arrow keys can be used to navigate through nested blocks.
